WordPressAuthenticator/Unified Auth/View Related/Password/PasswordViewController.swift

```diff
@@ -145,7 +145,9 @@ class PasswordViewController: LoginViewController {
             // is the actual error string. So check for "password" in the error string, and show the custom
             // error message. Otherwise, show the actual response error.
             var displayMessage: String {
+                // swiftlint:disable localization_comment
                 if nsError.localizedDescription.contains(NSLocalizedString("password", comment: "")) {
+                // swiftlint:enable localization_comment
```

Comment:
I didn't see the point in adding a comment explaining the intricate implementation details of why this localized string exists. It wouldn't be useful to translators.

I think we can all agree that error checking based on grepping a string is suboptimal but addressing it is out of scope right now.